sycl/include/sycl/ext/oneapi/experimental/group_sort.hpp

```diff
@@ -126,7 +125,7 @@ joint_sort([[maybe_unused]] Group group, [[maybe_unused]] Iter first,
   sorter(group, first, last);
 #else
   throw sycl::exception(
-      std::error_code(PI_ERROR_INVALID_DEVICE, sycl::sycl_category()),
+      std::error_code(UR_RESULT_ERROR_INVALID_DEVICE, sycl::sycl_category()),
```

Comment:
I think that neither `PI_ERROR_INVALID_DEVICE` nor `UR_RESULT_ERROR_INVALID_DEVICE` is correct here, it should be `sycl::errc::feature_not_supported`.

Reply:
This is outdated and has been corrected to use `sycl::errc::feature_not_supported`.